snippets/csharp/System.Linq/Queryable/AggregateTSource/queryable.cs

```diff
@@ -524,10 +530,10 @@ static void ContainsEx1()
         static void CountEx1()
         {
             // <Snippet22>
-            string[] fruits = { "apple", "banana", "mango",
-                                "orange", "passionfruit", "grape" };
+            string[] fruits = [ "apple", "banana", "mango",
+                                "orange", "passionfruit", "grape" ];
 
-            int numberOfFruits = fruits.AsQueryable().Count();
+            int numberOfFruits = fruits.Count();    // [CA1829, RCS1077] better to use "fruits.Length;"
```

Comment:
I'll stop commenting after this instance, but remove all these comments:

```suggestion
            int numberOfFruits = fruits.Count();  
```